queries.sql

```diff
@@ -33,3 +33,82 @@ WHERE name !== 'Gabumon'
 SELECT *
 FROM animals 
 WHERE weight_kg BETWEEN 10.4 AND 17.3
+
+-- Adding Species to the species column
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+-- Deleting the entire table and rolling back
+
+BEGIN;
+
+DELETE FROM animals;
```

Comment:
- [ ] You missed adding the below transaction, Please make sure to add it to the file and a screenshot of the results in the PR.

![image](https://github.com/skabeo/vet-clinic/assets/83803414/5df81bec-d891-4c7a-9e35-54c24cc7535e)